plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -900,10 +900,8 @@ public void testCreatePartitionedTableAs()
     }
 
     @Test
-    public void testColumnComments()
+    public void testAddColumnWithComment()
     {
-        // TODO add support for setting comments on existing column and replace the test with io.trino.testing.BaseConnectorTest#testCommentColumn
```

Comment:
BCT should provide sufficient coverage for adding column with comment. The test doesn't need to be Iceberg-specific.

cc @hashhar @ebyhr

Reply:
Yes, I filed #11486 yesterday.

Reply:
I moved the test to BaseConnectorTest.
If the column comment functionality is not available on `ADD COLUMN` we'll probably need a new entry in `TestingConnectorBehavior` enum

Reply:
> Yes, I filed #11486 yesterday.

thanks!

please make sure this Iceberg-specific test method gets removed as part of that issue

Reply:
I've largely shrinked the test to concentrate only on adding a column with comment.
However, there is a series of connectors (e.g.: Postgres, MsSQL) which do support adding columns, but ignore the comment provided.

I'll add a new testing behaviour `SUPPORTS_ADD_COLUMN_WITH_COMMENT` .